src/components/common/ColonyActions/ColonyActions.tsx

```diff
@@ -236,6 +236,23 @@ const ColonyActions = (/* { ethDomainId }: Props */) => {
         )}
         text="Test Unlock Token"
       />
+      <ActionButton
+        submit={ActionTypes.ACTION_MOVE_FUNDS}
+        error={ActionTypes.ACTION_MOVE_FUNDS_ERROR}
+        success={ActionTypes.ACTION_MOVE_FUNDS_SUCCESS}
+        transform={pipe(
+          mergePayload({
+            colonyAddress: colony.colonyAddress,
+            colonyName: colony.name,
+            fromDomain: colony.domains?.items.find((d) => d?.isRoot),
+            toDomain: colony.domains?.items.find((d) => !d?.isRoot),
+            amount: BigNumber.from(5).mul(BigNumber.from(10).pow(17)), // this is in wei
```

Comment:
I know this is for testing, but why is this a 17 decimals token ?

If it's a WEI, and it's a common 18 decimals token _(like we create normally through the colony client)_, this should be `pow(18)`

Reply:
It is 18 decimals. I wanted 0.5 of a token which needed to be converted into wei, but I couldn't call `BigNumber.from(0.5)...` because that was throwing an error. So I start with `5`, and instead decreased the exponent by one